websites/S/Songsterr/metadata.json

```diff
@@ -0,0 +1,28 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.7",
+	"author": {
+		"id": "263735224221827072",
+		"name": "AlperShal"
+	},
+	"service": "Songsterr",
+	"description": {
+		"en": "Songsterr is one of the most popular sites to find and play guitar or any instrument's tabs.",
+		"tr": "Songsterr gitar ya da herhangi bir çalgının tablarını bulmak ya da çalmak için en popüler sitelerden birisi."
+	},
+	"url": [
+		"www.songsterr.com",
+		"songsterr.com"
+	],
+	"version": "1.0.0",
+	"logo": "https://i.imgur.com/MXQP0Lq.png",
```

Comment:
[Logos](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md#logo) are required to have a resolution of 512x512px